tests/unit/api/bucketGetLocation.js

```diff
@@ -59,11 +68,12 @@ describe('getBucketLocation API', () => {
             });
         });
     });
-    [undefined, 'us-east-1'].forEach(location => {
+    ['us-east-1'].forEach(location => {
```

Comment:
Not sure if `undefined` should be tested since we are no longer giving the value as a parameter.